src/resources/resources/assets/waila/lang/ru_ru.json

```
@@ -247,25 +270,29 @@
   "config.waila.plugin_waila.icon_position_middle"              : "Посередине",
   "config.waila.plugin_waila.icon_position_bottom"              : "Внизу",
 
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+  "config.waila.plugin_wailax"                                  : "Waila Extra",
+  "config.waila.plugin_wailax.energy"                           : "Энергия",
+  "config.waila.plugin_wailax.energy.enabled_block"             : "Энергия блоков",
+  "config.waila.plugin_wailax.energy.enabled_entity"            : "Энергию сущностей",
+  "config.waila.plugin_wailax.energy.blacklist"                 : "Черный список",
+  "config.waila.plugin_wailax.progress"                         : "Процесс крафта",
+  "config.waila.plugin_wailax.progress.enabled_block"           : "Процесс крафта блоков",
+  "config.waila.plugin_wailax.progress.enabled_entity"          : "Процесс крафта сущностей",
+  "config.waila.plugin_wailax.progress.blacklist"               : "Черный список",
+  "config.waila.plugin_wailax.item"                             : "Предметы",
+  "config.waila.plugin_wailax.item.enabled_block"               : "Содержание блоков",
+  "config.waila.plugin_wailax.item.enabled_entity"              : "Содержимое сущностей",
+  "config.waila.plugin_wailax.item.nbt"                         : "Синхронизировать NBT данные",
+  "config.waila.plugin_wailax.item.max_height"                  : "Максимальная высота",
+  "config.waila.plugin_wailax.item.sort_by_count"               : "Сортировать по количеству",
+  "config.waila.plugin_wailax.item.blacklist"                   : "Черный список",
+  "config.waila.plugin_wailax.fluid"                            : "Жидкости",
+  "config.waila.plugin_wailax.fluid.enabled_block"              : "Жидкости в блоках",
+  "config.waila.plugin_wailax.fluid.enabled_entity"             : "Жидкости в сущностях",
+  "config.waila.plugin_wailax.fluid.display_unit"               : "Единицы измерения",
+  "config.waila.plugin_wailax.fluid.display_unit_millibuckets"  : "Милливедра (мВ)",
+  "config.waila.plugin_wailax.fluid.display_unit_droplets"      : "Дроплеты (Д)",
```

`d` is the unit for droplets, I don't think translating it makes sense. The actual unit shown is hardcoded anyway.
```suggestion
  "config.waila.plugin_wailax.fluid.display_unit_droplets"      : "Дроплеты (d)",
```